web-app-router/api/images/get_unverified_images.ts

```diff
@@ -22,10 +22,7 @@ export type ImageGetAllUnverifiedImagesResponse = {
  * @param req
  * @param res
  */
-export const handleGetAllUnverifiedImages = async (
-  req: NextApiRequest,
-  res: NextApiResponse,
-) => {
+export const GET = async (req: NextApiRequest, res: NextApiResponse) => {
```

Comment:
@andy-t-wang As I can see you import image handlers in legacy pages for now. I think we can keep these image handlers as it is in legacy folder for now. It should work, or am I missing something? 

Because app router's route handlers work a bit different and require a refectoring.